src/main/cms/cms_menu_osd.c

```diff
@@ -244,6 +244,8 @@ static const OSD_Entry menuOsdElemsEntries[] =
     OSD_ELEMENT_ENTRY("WIND HOR", OSD_WIND_SPEED_HORIZONTAL),
     OSD_ELEMENT_ENTRY("WIND VERT", OSD_WIND_SPEED_VERTICAL),
 
+    OSD_ELEMENT_ENTRY("GYRO TEMPERATURE", OSD_GYRO_TEMPERATURE),
```

Comment:
For now, we should add a generic temperature element that picks the best temperature source. If we need an individual indicator per temperature source we can add them later. I'd suggest you rename the user visible strings/constants as follows:

```suggestion
    OSD_ELEMENT_ENTRY("TEMPERATURE", OSD_TEMPERATURE),
```

Reply:
You are right, I forgot these when rebasing on the new framework.